core/src/main/java/org/bitcoinj/quorums/LLMQParameters.java

```diff
@@ -100,7 +100,7 @@ public static LLMQType fromValue(int value) {
 
         availableLlmqs.put(LLMQType.LLMQ_DEVNET, new LLMQParameters(LLMQType.LLMQ_DEVNET, "llmq_devnet",
                 12, 7, 6, 24, 2, 10,
-                18, 7, 2, 4, 6));
+                18, 7, 4, 4, 6));
```

Comment:
Originally in the `master` branch this value is 3, but previously was 2 and is now 4.  It is not clear if this will change again or return to 3 when this project is done.